common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -883,6 +918,25 @@ def answer_available(self):
 
         return False
 
+    def correctness_available(self):
+        """
+        Is the user allowed to see whether she's answered correctly?
+
+        Limits access to the correct/incorrect flags, messages, and problem score.
+        """
+        if self.show_correctness == SHOW_CORRECTNESS.NEVER:
+            return False
+        elif self.runtime.user_is_staff:
+            # This is after the 'never' check because admins can see correctness
```

Comment:
@pomegranited Can you clarify why this else is after checking SHOW_CORRECTNESS.NEVER?